socioeconomic-differences/socioeconomic-differences.md

```diff
@@ -12,66 +15,71 @@ A person's socioeconomic status can inform their identity, as it affects the com
-
+    1.Do not spend a lot of time explaining each point, rather try to fit the points effectively within the given time frame by explaining each points briefly but thoroughly,however try your best to include the suggestions and remarks made by students during the course of discussion.Students can be divided into groups and each group can briefly discuss on topics learnt such as:-
+    a. Economic and social constraints
+    b. Methods of tackling pressure and problems due to professional class transformation etc.
 
-### Independent Practice
+    2.Students can engage in an informative activity where the ones who identify themselves as "Class Straddler" can narrate the "good" and the "bad" experiences they have experienced so far.
 
-Class does this thing themselves with specific additional items. This could be alone, with a partner, or small group; but the idea is that it's less guided, more independent.
 
-### Challenge
+    3. The participants may feel reluctant to initially share their experience.The instructor should try to encourage them kindly and carefully without asserting any obligation.
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
+    4. The students can recount the examples of local professionals in the area,who too had blue-collar lifestyle but successfully transitioned to white-collar job.Also do mention the struggles they faced(however do not repeat or elaborate problems mentioned above, rather keep it more personal and healthy discussion)
 
+### Check for Understanding(37-40 minutes)
 
-### Check for Understanding
-
-Some ideas: have apprentices summarize to each other, make a cheat sheet, take a quiz, do an assignment, or something else that helps them self-assess their understanding. This exercise should help apprentices determine whether they've met the outline objectives, or if they need to review.
+    1. A discussion can be held in the end where the participants summarize each other.
+    2. Ask the students to read the first 30 pages of "Blue-Collar Roots, White-Collar Dreams" by Alfred Lubrano,and narrate the essence of what they have read and understood so far.  
+    3. Ask the students  specifically regarding the problems one faces in order to achieve promotion as "Class Straddlers".Ask them what they think are the appropriate measures to tackle the mentioned problems.
+     
 
 ### Supplemental Materials
-- [example website](https://example.com) - Write a very short description of how to use this optional resource.
-- [Other example website](https://otherexample.com) - Write a very short description of how to use this optional resource.
+1.[link](https://genprogress.org/straddling-college-and-working-class/) - The following article derives a narrative about the problems working-class students face in elite schools and furthermore the effect of the problem may hinder their jobs too.
```

Comment:
Please give these two links the names of the resource and author rather than “link”.